sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -257,7 +257,8 @@ static HostColumnVector.DataType convertFrom(DataType spark, boolean nullable) {
       }
       return new HostColumnVector.StructType(nullable, children);
     } else if (spark instanceof BinaryType) {
-      return new HostColumnVector.ListType(nullable, convertFrom(DataTypes.ByteType, false));
+      return new HostColumnVector.ListType(
+          nullable, new HostColumnVector.BasicType(nullable, DType.UINT8));
```

Comment:
Bytes in the binary list-of-bytes should not be nullable.
```suggestion
          nullable, new HostColumnVector.BasicType(false, DType.UINT8));
```

Reply:
ah makes sense.